tests/driver/api.test.ts

```diff
@@ -955,7 +987,7 @@ describe('Mongoose Model API level tests', async () => {
             const collection = collections.find(collection => collection.name === 'vector');
             assert.ok(collection, 'Collection named "vector" not found');
             assert.deepStrictEqual(collection.options, {
-                vector: { dimension: 2, metric: 'cosine' }
+                vector: { dimension: 2, metric: 'cosine', sourceModel: 'other' }
```

Comment:
Is this a correct behavior? this might require a change on the Data API side not to return the `sourceModel` field when a collection is not configured with?

Reply:
This seems to be expected behavior re: https://github.com/stargate/data-api/pull/1606/files: vector option now always includes `sourceModel` in the result, even if $vectorize isn't used. See `happyWithMetricOnly` test in [this file](https://github.com/stargate/data-api/pull/1606/files#diff-198e2cf5fb9b9512af281dd7cde1dca90bdabc40d289561d8fd0c41e9d5baa5a).